lib/game/widget_builder.dart

```diff
@@ -1,157 +1,198 @@
 import 'package:flutter/widgets.dart';
+import 'package:flutter/gestures.dart';
+
+import './gesture_detector.dart' as flame_detector;
 import '../gestures.dart';
 import 'embedded_game_widget.dart';
 import 'game.dart';
 
+bool _hasGestureDetectors(Game game) =>
```

Comment:
Do all these detectors not inherit from some common baseclass?

Reply:
Nope, they are mixins, mixins do not have inheritance :(